src/main/java/jenkins/plugins/office365connector/WebhookJobPropertyDescriptor.java

```diff
@@ -73,7 +73,16 @@ public WebhookJobProperty newInstance(StaplerRequest req, JSONObject formData) t
                     webhooks.addAll(req.bindJSONToList(Webhook.class, webhooksArrayData));
                 } else {
                     JSONObject webhooksObjectData = (JSONObject) webhooksData;
-                    webhooks.add(req.bindJSON(Webhook.class, webhooksObjectData));
+                    Boolean isCompactNotificationOn = webhooksObjectData.getBoolean("compactNotification");
+                    Webhook webhook = req.bindJSON(Webhook.class, webhooksObjectData);
+                    if (isCompactNotificationOn==true)
+                    {
```

Comment:
formatting...

Reply:
do you have any formatting document where I can see all the rules?

Reply:
opening bracket is just next to the condition - this is Java standard, quite well documented if you Google for Java formatting standard - or just simple use Java editor with Java formatting

Reply:
Thanks, already figured it out thanks to @casz